consensus/processor_test.go

```diff
@@ -1074,3 +1099,11 @@ func mustSetup2Actors(t *testing.T, balance1 *types.AttoFIL, balance2 *types.Att
 	})
 	return addr1, act1, addr2, act2, st, mockSigner
 }
+
+func mustCreateMinerWithOwner(ctx context.Context, require *require.Assertions, st state.Tree, vms vm.StorageMap, minerAddr, minerOwner address.Address) (cid.Cid, *actor.Actor) {
```

Comment:
name of this function implies to me that it creates the owner as well. looking at some of the tests that use it, seems like they might expect that it does